tools/deb/test/envoy_local.json

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
is this the shortest local config we can get, seems pretty long... and how will get updated?